templates/qcbm-optimization.yaml

```diff
@@ -33,15 +33,15 @@ spec:
             max_val = {{inputs.parameters.max-val}}
             seed = {{inputs.parameters.seed}}
 
-            parameters = generate_random_initial_params(n_qubits, n_layers=n_layers, 
+            parameters = generate_random_initial_params(n_qubits, n_layers=n_layers,
```

Comment:
This template can be deleted and replaced in the example workflow by the `generate-random-ansatz-params` template located in `z-quantum-core/templates/circuit.yaml`. Using said template, you can pass either `ansatz-specs` or `number-of-params` explicitly.

Reply:
See [here](https://github.com/zapatacomputing/z-quantum-qcbm/pull/12/commits/8e2d63ead3fb4395b62119406b7627e9231d47a0) and [here](https://github.com/zapatacomputing/z-quantum-qcbm/pull/12/commits/deb336876d20767c5d7808241a4a0a9f02253708)